api_functional_v2_test.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package minio_test
+package minio
 
```

Comment:
I found this https://golang.org/src/bytes/export_test.go which exports variables for testing purpose as written, but I also found this too: https://golang.org/src/log/log_test.go and  https://golang.org/src/log/example_test.go which implies in my sense that there are unit tests for internal code, and functional tests for the API.. It doesn't seem that there are any standard anyway.. besides, minPartSize should be private so the developer won't be able to modify it especially during operation.